### PR TITLE
Fix #23451: add -A command

### LINKED ISSUE #23451: Implement -A command like r2 -A

### PR DESCRIPTION
**Pull request by astralia**
- [ ] Mark this if you consider it ready to merge
- [*] I've added tests (optional)
- [ ] I wrote some lines in the [book](https://github.com/radareorg/radare2book) (optional)

**Description**

Related to [this issue](https://github.com/radareorg/radare2/issues/23451).
Also added '-' to ? help message

### CODE REVIEW

File: libr/core/cmd.c [hunk 1]
```diff
@@ -2071,6 +2072,21 @@ static int cmd_stdin(void *data, const char *input) {
...
 			break;
+		case 'A': // -A
+			if (*arg == '?') {
+				r_core_cmd_call (core, "aaa?");
+			} else {
+				if (R_STR_ISEMPTY (arg)) {
+					r_core_cmd_call (core, "aaa");
+				} else if (*arg == 'A') {
+					r_core_cmd_call (core, "aaaa");
+				} else if (strcmp(arg, "AA")) {
```
Comment (trufae) [inline libr/core/cmd.c:2083]:
```suggestion
				} else if (!strcmp (arg, "AA")) {
```

File: libr/core/cmd.c [hunk 2]
```diff
@@ -2071,6 +2072,21 @@ static int cmd_stdin(void *data, const char *input) {
 				r_core_cmdf (core, "e %s", arg);
 			}
 			break;
+		case 'A': // -A
+			if (*arg == '?') {
+				r_core_cmd_call (core, "aaa?");
+			} else {
+				if (R_STR_ISEMPTY (arg)) {
+					r_core_cmd_call (core, "aaa");
+				} else if (*arg == 'A') {
```
Comment (trufae) [inline libr/core/cmd.c:2081]: better use !
```suggestion
				} else if (!strcmp (arg, "A")) {
```

**Review by trufae — CHANGES_REQUESTED** [review]

File: test/db/tools/r2 [hunk 3]
```diff
@@ -125,3 +125,67 @@ EXPECT_ERR=<<EOF
...
 RUN
+
+NAME=radare2 aaa
+FILE=malloc://123
+CMDS=<<EOF
+-A
+?ee --
+aaa
+?ee --
+r2 -qA malloc://123
```
Comment (trufae) [inline test/db/tools/r2:136]:
```suggestion
r2 -e scr.color=0 -qA malloc://123
```

try this hopefully that will disable the colors

File: libr/core/cmd.c (hunk 1 above)
Comment (astralia) [reply libr/core/cmd.c]: Fixed

File: libr/core/cmd.c (hunk 2 above)
Comment (astralia) [reply libr/core/cmd.c]: fixed

File: test/db/tools/r2 (hunk 3 above)
Comment (astralia) [reply test/db/tools/r2]: Fixed but keeps displaying weird stuff with the msg `INFO: Analyze function calls (aac)`

**Pull request merged**